packages/kit/src/core/adapt/plugins.js

```diff
@@ -0,0 +1,13 @@
+import { join } from 'path';
+
+/** @type {import('@sveltejs/kit/adapter').AppResolver} */
+export function appResolver() {
+	return {
+		name: '@sveltejs/esbuild-plugin-app-resolver',
+		setup(build) {
+			build.onResolve({ filter: /@sveltejs\/kit\/app/ }, (args) => ({
+				path: join(args.resolveDir, '../output/server/app.js')
```

Comment:
can this be configurable? I am finding it difficult to visualise this process in my mind

Reply:
nvm, just getting confused.